README.md

```diff
@@ -108,7 +108,7 @@ Please be respectful and be aware that this is maintained with our free time!
 
 for community support or general questions.
 Feel free to post on our subreddit <https://www.reddit.com/r/pivpn/>
-You can also join #pivpn <ircs://freenode/pivpn> on freenode in IRC
+You can also join #pivpn on [libera.chat](https://libera.chat) IRC network
```

Comment:
Yeah maybe. I merged the latest master state before creating a branch. The branch should have been clean since I commited the change a few hours later.